pkgs/applications/window-managers/taffybar/default.nix

```diff
@@ -15,5 +15,13 @@ in stdenv.mkDerivation {
       --set NIX_GHC "${taffybarEnv}/bin/ghc"
   '';
 
-  inherit (haskellPackages.taffybar) meta;
+  # For hacking purposes
+  passthru.env = taffybarEnv;
+
+  meta = taffybar.meta // {
+    maintainers = with lib.maintainers; [ rvl ];
```

Comment:
It'd be preferrable if you added yourself in `haskellPackages` already: You can do that by adding an to `package-maintainers` in `pkgs/development/haskell-modules/configuration-hackage2nix/main.yaml` (and rerunning `./maintainers/scripts/haskell/regenerate-haskell-packages.nix`).

Reply:
Aha right, simple, fixed now.